src/ray/raylet/test/local_object_manager_test.cc

```diff
@@ -380,14 +380,14 @@ TEST_F(LocalObjectManagerTest, TestSpillObjectsOfSize) {
   }
   manager.PinObjects(object_ids, std::move(objects));
 
-  int64_t num_bytes_required = manager.SpillObjectsOfSize(total_size / 2);
+  int64_t num_bytes_required = manager.SpillObjectsOfSize(total_size / 2, total_size / 2);
```

Comment:
Can you add a new unit test to check the return value of `SpillObjectsOfSize` when `num_bytes != min`?